pkgs/development/libraries/qt-6/README.md

```diff
@@ -0,0 +1,81 @@
+# Qt 5 Maintainer's Notes
```

Comment:
qt6 ...

Reply:
Yes, I agree this whole file is now irrelevant.